src/chart/radar/radarLayout.js

```diff
@@ -38,8 +39,19 @@ export default function (ecModel) {
 
         data.each(function (idx) {
             // Close polygon
-            points[idx][0] && points[idx].push(points[idx][0].slice());
+
+            var _idx = findFirstNotNaNPoint()
```

Comment:
ECharts use camel cases, and it's better to name it something meaningful, like `firstId`.

Reply:
Add `;` at the end of each line. Also note other places.